tests/datastructures/test_masking.py

```diff
@@ -7,7 +7,7 @@
 
 class TestMask(TestCase):
 
-    NETWORK_SIZE = 1024
+    NETWORK_SIZE = 200
```

Comment:
I'm not sure whether it's good or bad idea, but it's totally unrelated to WTCT...

Reply:
yup, it was terribly slow, thought you're welcome speedup. shall I revert?

Reply:
Yes. It would be nice to have it as a separate PR.

Reply:
ok

Reply:
see #341